docs/node-observ-lib/alerts.libsonnet

```diff
@@ -0,0 +1,420 @@
+{
+  new(this): {
+    groups: [
+      {
+        name: if this.config.uid == 'node' then 'node-exporter-filesystem' else this.config.uid + '-linux-filesystem-alerts',
+        rules: [
+          {
+            alert: 'NodeFilesystemSpaceFillingUp',
+            expr: |||
+              (
+                node_filesystem_avail_bytes{%(filteringSelector)s,%(fsSelector)s,%(fsMountpointSelector)s} / node_filesystem_size_bytes{%(filteringSelector)s,%(fsSelector)s,%(fsMountpointSelector)s} * 100 < %(fsSpaceFillingUpWarningThreshold)d
+              and
+                predict_linear(node_filesystem_avail_bytes{%(filteringSelector)s,%(fsSelector)s,%(fsMountpointSelector)s}[6h], 24*60*60) < 0
+              and
+                node_filesystem_readonly{%(filteringSelector)s,%(fsSelector)s,%(fsMountpointSelector)s} == 0
+              )
+            ||| % this.config,
+            'for': '1h',
```

Comment:
Not something that has to be changed, but I'm curious why these alerts are 1 hour triggers?

Reply:
TBH, something that was set upstream, I would rather avoid such large intervals myself. I think that can be tuned separately.

Reply:
Ahh makes sense, something we can iterate on then 😄

Reply:
I think logic here to avoid false positives due to backups/some short activity that writes to disk.  So only if your system's disk filling up for extended period of time - then alert.